app/main.py

```diff
@@ -1,3 +1,58 @@
+import json
+import datetime
+
+
 def shop_trip():
-    # write your code here
-    pass
+    with open('app/config.json', 'r') as json_file:
+        json_data = json.load(json_file)
+
+    fuel_price = json_data["FUEL_PRICE"]
+    customers = json_data["customers"]
+    shops = json_data["shops"]
+
+    for c in customers:
```

Comment:
what is `c`?)

Reply:
`c` is short for `customer` )) otherwise flake8 test was failed ))